### DISCUSSION TITLE
How to modify the default theme color of Arco Design Pro using Rsbuild

### DISCUSSION THREAD

**Discussion by JonasVon** [web-infra-dev/rsbuild#3475] [Q&A]
根据Arco Design Pro官网的描述，只需要修改lessOptions即可：
<img width="1630" alt="image" src="https://github.com/user-attachments/assets/8b391229-0b5f-489b-85e2-4d188ecfe224">
在rsbuild中修改lessOptions 没有生效，rsbuild配置如下：
pluginLess({
      lessLoaderOptions: {
        lessOptions: {
          modifyVars: {
            "--primary-6": "#2590FF",
            "--arcoblue-6": "#2590FF",
          },
        }
      }
    }),

跪求大佬们帮忙看看如何解决，跪谢🙇。

**Comment by 9aoy** [comment]
I think you can use `rsbuild inspect` to check whether less options are added normally in the final output rspack configuration.  

https://rsbuild.dev/zh/guide/basic/cli#rsbuild-inspect

**Reply by JonasVon** [reply]
This issue has been resolved in V1.0.4, thanks.

**Comment by JonasVon** [comment]
This issue has been resolved in V1.0.4, please upgrade @rsbuild/core to V1.0.4
